src/services/user/hooks/index.js

```diff
@@ -433,7 +440,7 @@ exports.after = {
 	get: [
 		decorateAvatar,
 		decorateUser,
-		computeProperty(User.userModel, 'getPermissions', 'permissions'),
+    populateRoles,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5229889)